book/src/prerequisites.md

```diff
@@ -1,21 +1,73 @@
-# Installation
+# Prerequisites
+
+There are a few recommended ways to set up your working station in order to develop `gtk-rs` applications.
+Let us go through them one by one.
+
+## Cargo
+
+[Cargo](https://doc.rust-lang.org/cargo/index.html) is Rust's build system and package manager.
+If following the book is all you care about, using only Cargo will work fine for you.
 
 Let us begin by installing all necessary tools.
-First follow the instructions on the [GTK website](https://www.gtk.org/docs/installations/) to install GTK 4.
+First follow the instructions on the [GTK website](https://www.gtk.org/docs/installations/) in order to install GTK 4.
 Then install Rust with [rustup](https://rustup.rs/).
 
 Now create a new project by executing:
 ```bash
 cargo new my-gtk-app
 ```
 
-Add the following lines to your `Cargo.toml` and you are ready to go!
+Add the following lines to your `Cargo.toml`.
 ```toml
 [dependencies.gtk]
 version = "0.1"
 package = "gtk4"
 ```
 
-# Buildsystem
+Now you can run your application by executing:
+```bash
+cargo run
+```
+
+## Cargo + Meson
+
+Cargo is *nearly* enough, but it is not well suited for handling resources such as icons or UI files.
```

Comment:
I would add handling translation through gettext as well

Reply:
Kind of included in "GTK functionality".

I can replace "GTK functionality" with actual examples, but I wonder if that might be confusing for total GTK beginners

Reply:
I don't see any mention of "GTK functionality"? I wouldn't go too specific in details but still give few reasons of what Meson would simplify for the developer.

Reply:
Sure, which ones would you deem the most important?

Reply:
It simplifies a bunch of things: 
- Translations: Simplify a bunch of stuff through the gettext module like translating appdata/desktop files or installing translations
- Building/installing GResources (UI files and all)
- Installing files such as icons

Reply:
Also settings, the schemas